scene/resources/theme.cpp

```diff
@@ -1240,6 +1240,11 @@ void Theme::get_type_list(List<StringName> *p_list) const {
 		types.insert(E.key);
 	}
 
+	// Variations
```

Comment:
```suggestion
	// Variations.
```

For consistent style.

Reply:
Sorry about that. it should be fixed now